extension/src/sidebar-outer.css

```diff
@@ -1,7 +1,20 @@
-#promnesia-sidebar.promnesia {
+/* Resets for sites */
+body.promnesia-sidebar-active {
+    transform: none;
```

Comment:
oh now I see why it's here https://github.com/karlicoss/promnesia/issues/102#issuecomment-1114237352
can you add a link to the issue please? because otherwise it's not obvious what this code is for :)

Reply:
ok